data-tck/src/main/java/io/micronaut/data/tck/entities/PatientDto.java

```diff
@@ -0,0 +1,31 @@
+package io.micronaut.data.tck.entities;
+
+import io.micronaut.core.annotation.Introspected;
+
+@Introspected
+public class PatientDto {
+
+    private final String name;
+
+    private final String history;
+
+    private final String doctorNotes;
+
+    public PatientDto(String name, String history, String doctorNotes) {
+        this.name = name;
+        this.history = history;
+        this.doctorNotes = doctorNotes;
```

Comment:
Mapped entity Patient does not have doctorNotes field and DTO does have, tests with this entity were failing/issue reproducible before fix was checked in.